funlib/persistence/arrays/array.py

```diff
@@ -118,8 +137,8 @@ def uncollapsed_dims(self, physical: bool = False) -> list[bool]:
         if physical:
             return [
                 x
-                for x, c in zip(self._uncollapsed_dims, self._metadata.axis_names)
-                if not c.endswith("^")
+                for x, t in zip(self._uncollapsed_dims, self._metadata.types)
```

Comment:
Is time not a physical dim? What is "uncollapsed" in this context? I went to look at the docs to see what we defined as physical and there aren't any 😬 so let's at least put a docstring with a definition

Reply:
Yeah its not well defined because Jan and I discussed treating time a bit special.
If you have `t,z,y,x`, you could pass in a 4D Roi and get a specific `t,z,y,x` slice, or you could pass in a 3D Roi and get a specific `z,y,x` slice with all time points.
I think I might leave that for future implementation if desired and for now simply treat time the same as space everywhere.

Reply:
Done